data/strings/strings.txt

```diff
@@ -22858,6 +22858,141 @@
     zh-Hans = 加载地图
     zh-Hant = 載入地圖
 
+  [pref_tts_speedcams_auto]
+    tags = android
+    en = Warn about speedcams if there is a risk of exceeding the speed limit
```

Comment:
Thanks! If I understood correctly, the UI now looks like this:

[Speed cameras]
[Warn about speedcams if there is a risk of exceeding the speed limit]

As there is already [Speed cameras] title, it's better to simplify all strings by removing "about speedcams":

[Speed cameras]
[Warn if there is a risk of exceeding the speed limit]

Reply:
This is how the UI looks like right now...
![and_01](https://github.com/organicmaps/organicmaps/assets/25888296/8be938e9-a465-4e3b-8d34-137c7560c7a0)


And this is after the removal of "about speedcams" string:
![and_02](https://github.com/organicmaps/organicmaps/assets/25888296/963fb554-7d78-4f05-8288-c40f6d7c67fe)

Reply:
The main issue here is going to be the removal of the "about speedcams" in the strings of the different languages...

I'm going to try to do it myself.. It should not be hard for most of the languages, but for some of them (specially asian or arabic), my changes could be a really big mess...

We can request the help of the translation community here...

Reply:
I've updated the commit with the removal of "about speedcams" in the strings for all languages.

For all languages, except these 4 (which  I find quite hard to modify without knowledge...):
- ja (Japanese)
- th (Thai)
- zh-Hans (Chinese simplified)
- zh-Hant (Chinese traditional)